scripts/dockerfiles/Dockerfile.metrics

```diff
@@ -18,16 +20,16 @@ RUN go mod download
 COPY . .
 RUN make build-metrics
 
-FROM amazonlinux:2
+FROM $docker_arch/amazonlinux:2
 RUN yum update -y && \
-    yum clean all
+	yum clean all
```

Comment:
do you need to put this in here? (you added it to the release Dockerfile below)

```
ENV GOARCH=$arch
```

Reply:
Good catch, yes, this'll need to be plumbed in (in the `builder` stage). Will update.

---

A side note:

With a static build, I think that the CNI metrics helper's container image can be shrunk down into a final image of:

```dockerfile
FROM scratch
# Copy our bundled certs to the first place go will check: see
# https://golang.org/src/pkg/crypto/x509/root_unix.go
COPY ./misc/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
COPY --from=builder /go/src/github.com/aws/amazon-vpc-cni-k8s/cni-metrics-helper /app
```

That's a change for another time though.